scripts/variantstore/beta_docs/gvs-workspace-description.md

```diff
@@ -61,20 +62,18 @@ For troubleshooting or questions, contact the [Broad Variants team](mailto:varia
 
 ## Running the workflow
 
-The `GvsBeta` workflow in the GVS beta workspace is pre-configured to use 10 sample GVCF files in the workspace Data tab. To run:
+The `GvsBeta` workflow in the GVS beta workspace is pre-configured to use 10 sample reblocked GVCF files in the workspace Data tab. To run the example data:
 
 1. **Select the workflow** from the Workflows tab.
 1. Select the 'Run workflow with inputs defined by file paths' radio button.
 1. Configure the workflow inputs.
     1. Enter a **name for the callset** as a string with the format “*CALLSET_NAME*” for the `call_set_identifier` variable. This string is used as to name several variables and files and should begin with a letter. Valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”.
     1. Enter the name of your **BigQuery dataset** as a string with the format “*DATASET_NAME*” for the `dataset_name` variable. Valid characters include A-z, 0-9, “,”, “-”, and “_”.
-    1. Enter a GCS path for `extract_output_gcs_dir`, which is where the callset VCFs, VCF indexes and interval lists will be copied.
     1. Enter the name of the **GCP project** that holds the BigQuery dataset as a string with the format “*PROJECT_NAME*” for the `project_id` variable.
-    1. Ensure that the (optional) `is_wgs` parameter is set to false.
-    1. Check the workspace Data tab, and fill out the below inputs according to the data column names that contain the values you want to use:
-        - `sample_id_column_name`: a unique identifier for each sample (the workflow will fail if there are any duplicates)
-        - `vcf_files_column_name`: the path to the sample VCF file
-        - `vcf_index_files_column_name`: the path to the sample VCF index file
+    2. Update the name of the **column of reblocked GVCFs** in your samples table in the workspace Data tab for the `vcf_files_column_name` variable with the format "*COLUMN_NAME*" if it is different from the default. If you used the ReblockGVCF workflow in the workspace without modification, this will be the default string _"reblocked_gvcf"_.
```

Comment:
dont we want these to all be "1" so that markdown deals with it accordingly?

Reply:
I thought so too, but when I do preview in github it looks correct. And I had already updated the workspace description and it looks ok there too https://app.terra.bio/#workspaces/gvs-prod/Genomic_Variant_Store_Beta